src/wasm/wat-parser.cpp

```diff
@@ -282,6 +281,11 @@ struct GlobalType {
   Type type;
 };
 
+struct TypeUse {
```

Comment:
Maybe add a comment here? Puzzling over this I don't have a good guess as to what a "type use" is that contains a type and a vector of name/type pairs.

Reply:
Will add a comment about how it is used, but it comes from here: https://webassembly.github.io/spec/core/text/modules.html#type-uses

Reply:
As, so the vector is the parameter names? Makes sense.